modules/sony/display/cbx.rb

```diff
@@ -0,0 +1,61 @@
+module Sony; end
+module Sony::Display; end
+
+# Documentation: https://docs.google.com/spreadsheets/d/1F8RyaDqLYlKBT7fHJ_PT7yHwH3vVMZue4zUc5McJVYM/edit?usp=sharing
+
+class Sony::Display::CBX
+    include ::Orchestrator::Constants
+    include ::Orchestrator::Transcoder
+
+    # Discovery Information
+    tcp_port 4999
+    descriptive_name 'Sony CBX Display'
+    generic_name :Display
+
+    def on_load
+        on_update
+    end
+
+    def on_update
+    end
+
+    def connected
+        schedule.every('60s') { power? }
+    end
+
+    def disconnected
+        schedule.clear
```

Comment:
The only scheduled item looks to be the power poll. As well as providing status info, this looks to also act as a heartbeat. By stopping this schedule, if a device drops offline it won't attempt to reconnect without a `power` command being explicitly sent (so will continue to show offline in diagnostics).

You may want to keep this in place, or still clear it, but pop in a new schedule the fires at a longer interval (say every 5 mins) until the device connects again.